projects/uitdatabank/docs/entry-api/shared/videos.md

```diff
@@ -1,8 +1,89 @@
-Coming soon.
-
-<!--
-  @todo
-  - Explain supported types of videos
-  - Explain how to add/update/remove videos from events/places
-  - Explain how to embed on your website
--->
+# Videos
+
+Videos make your events or places stand out from the crowd and more attractive to visitors.
+
+Online calendars can show an embedded video player, so results with videos are more likely to be checked out by visitors.
+
+In this guide you will learn how to link videos to your events and places. Additionally, you'll learn how to update or remove the videos afterward.
+
+## Scope
+
+Videos can linked to:
+
+* ✅ [Events](../events/introduction.md)
+* ✅ [Places](../places/introduction.md)
+* ❌ [Organizers](../organizers/introduction.md)
+
+## Supported video sites
+
+Only videos from [YouTube](https://www.youtube.com/) or Vimeo are supported.
```

Comment:
Vimeo does send some javascript (I suspect to block embedding) when using `[Vimeo](https://vimeo.com)`, and it breaks the page. So for consistency, it might be better to remove the YouTube-link. What do you think @corneelwille ?

Reply:
```suggestion
Only videos from YouTube or Vimeo are supported.
```

Reply:
@JonasVHG okay! See suggested commit